packages/mobile/src/services/query-client.ts

```diff
@@ -5,12 +5,15 @@ import { env } from 'app/services/env'
 export const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
+      // Tan-query auto retries on errors like 404s. This doesnt fit with our backend design so we've disabled retries by default. They can be re-enabled at the query level if needed.
+      retry: false,
```

Comment:
instead let's go for this solve

https://github.com/TanStack/query/discussions/372#discussioncomment-7807126

i think it'll be more durable in the long run

Reply:
Most if not all of these should be `ResponseError` types which will make getting the statuses even easier. I think we should not retry for all 4xx errors to start